### ISSUE TITLE
[Bug]: `<!--#-->` comments are not removed

### ISSUE THREAD

**Issue by samualtnorman** [terser/html-minifier-terser#155]
### What happened?

`removeComments: true` removes all comments except those that contain `#`.

If this is intentional behaviour, it would be neat if we had a flag to override it.

### Version

v7.2.0

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari

### Link to reproduce

https://stackblitz.com/edit/stackblitz-starters-trumlr?file=index.js

### Willing to submit a PR?

No

**Comment by DanielRuf** [comment]
> If this is intentional behaviour, it would be neat if we had a flag to override it.

Yes, see the `ignoreCustomComments` setting, that you can change.

ignoreCustomComments | Array of regex'es that allow to ignore certain comments, when matched | [ /^!/, /^\s*#/ ]
-- | -- | --

So in your case removing `, /^\s*#/` should work.

https://github.com/terser/html-minifier-terser?tab=readme-ov-file#options-quick-reference

**Comment by DanielRuf** [comment]
> If this is intentional behaviour, it would be neat if we had a flag to override it.

Yes, see the `ignoreCustomComments` setting, that you can change.

https://github.com/terser/html-minifier-terser?tab=readme-ov-file#options-quick-reference

ignoreCustomComments | Array of regex'es that allow to ignore certain comments, when matched | [ /^!/, /^\s*#/ ]
-- | -- | --

So in your case removing `, /^\s*#/` should work.
